packages/pagination/src/locales/sv/messages.po

```diff
@@ -0,0 +1,64 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2023-11-28 09:50+0100\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: sv\n"
+"Project-Id-Version: 2853b3c41d7b7a59e7cd64c0153b540a\n"
+"Report-Msgid-Bugs-To: \n"
+"Last-Translator: \n"
+"Language-Team: Swedish\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Crowdin-Project: 2853b3c41d7b7a59e7cd64c0153b540a\n"
+"X-Crowdin-Project-ID: 141\n"
+"X-Crowdin-Language: sv\n"
+"X-Crowdin-File-ID: 15506\n"
+
+#. Default screenreader message for first page link in the pagination component
+#. js-lingui-explicit-id
+#: packages/pagination/src/FirstPage.tsx:38
+msgid "pagination.aria.first-page"
+msgstr "Förstasida"
+
+#. Suffix added at the end of icon titles when img semantics are lost on an html element
+#. Suffix added at the end of icon titles when img semantics are lost on an html element
+#. Suffix added at the end of icon titles when img semantics are lost on an html element
+#. js-lingui-explicit-id
+#: packages/pagination/src/FirstPage.tsx:44
+#: packages/pagination/src/NextPage.tsx:48
+#: packages/pagination/src/PrevPage.tsx:48
+msgid "pagination.aria.icon-suffix"
+msgstr "ikon"
+
+#. Default screenreader message for next page link in the pagination component
+#. js-lingui-explicit-id
+#: packages/pagination/src/NextPage.tsx:42
+msgid "pagination.aria.next-page"
+msgstr "Nästa sida"
+
+#. Default message for current page label in the pagination component
+#. js-lingui-explicit-id
+#: packages/pagination/src/CurrentPage.tsx:21
+msgid "pagination.label.current-page"
+msgstr "Sidan {currentPage}"
+
+#. Default screenreader message for page link in the pagination component
+#. js-lingui-explicit-id
+#: packages/pagination/src/Page.tsx:39
+msgid "pagination.aria.page"
+msgstr "Sidan {currentPage}"
```

Comment:
Is it correct that the word "Sidan" is used here (definite form) and not "Sida" (indefinite form like in the other translations)?